libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -128,7 +129,7 @@ keyHashForMultisig :: forall era. Proof era -> KeyHash 'Witness
 keyHashForMultisig pf = hashKey . vKey $ keysForMultisig pf
 
 simpleScript :: forall era. Scriptic era => Proof era -> Script era
-simpleScript pf = fromNativeScript $ allOf [require @era (keyHashForMultisig pf)] pf
+simpleScript pf = fromNativeScript $ Scriptic.allOf [require @era (keyHashForMultisig pf)] pf
```

Comment:
Need to make uses of `anyOf` and `allOf` qualified to avoid a name clash from functions of the same name in `microlens`.